### PR TITLE
Fix issue with missing required field default value in Navbar JSON schema.

### PR DESCRIPTION
**Pull request by YiHuangIX**
This is fix for #129 Demo code /docs Fetch error Internal Server Error /openapi.json 

**Issue:**
when generating openapi schema for fastui component, for navibar component, default value for key = "required" is missing 
**Solution:**
Add default value when generating schema

Note:
In docs some api point loading still very slow due to known issue: see https://github.com/pydantic/FastUI/issues/58

### CODE REVIEW

**Comment by samuelcolvin** [comment]
We should add a test.

Please have a go, if you can't I'll add one.

**Comment by YiHuangIX** [comment]
Added test for /docs endpoint, please review. 

PS: I have trouble doing pre-commit check locally so I uninstall it, and then so many lint issue.... If I turn on pre-commit it report file:
```
[ERROR] Your pre-commit configuration is unstaged.
`git add .pre-commit-config.yaml` to fix this.
```
I do'nt know what to do then...

**Comment by samuelcolvin** [comment]
@YiHuangIX contributing to open source (and software development in general) takes a fair git of persistence and willingness to solve problems yourself. You shouldn't just post every error you get to the project in the hope and expectation that someone else will tell you how to fix it. Your time is not more valuable than mine!

To solve pre-commit, you can:
* add the `-n` flag to skip pre-commit completely
* set the `SKIP=no-commit-to-branch` env variable when commiting
* (the best answer, and something you should do anyway) not use your main branch creating pull requests

**Comment by samuelcolvin** [comment]
thanks so much.

**Comment by YiHuangIX** [comment]
> @YiHuangIX contributing to open source (and software development in general) takes a fair git of persistence and willingness to solve problems yourself. You shouldn't just post every error you get to the project in the hope and expectation that someone else will tell you how to fix it. Your time is not more valuable than mine!
> 
> To solve pre-commit, you can:
> 
> * add the `-n` flag to skip pre-commit completely
> * set the `SKIP=no-commit-to-branch` env variable when commiting
> * (the best answer, and something you should do anyway) not use your main branch creating pull requests

Thank you! For all your comments and feedbacks... I am still new to opensource communities, I am happy to work with leaders like you.

File: demo/tests.py [hunk 1]
```diff
@@ -64,3 +64,14 @@ def test_menu_links(url: str):
 
 
 # TODO tests for forms, including submission
+
+
+def test_doc():
+    r = client.get('/docs')
```
[the remark this answers is not in the document]
Comment (YiHuangIX) [reply demo/tests.py]: Saw your new test scripts. Well done!

**Pull request merged**